src/app/services/api.service.ts

```diff
@@ -73,8 +73,7 @@ export class ApiService {
 
   put(pk, obj, queryParamsObject = null): Promise<any> {
     let queryString = this.generateQueryString(queryParamsObject);
-    // This is not a mistake, we use post to edit in AWS.
-    return this.http.post<any>(`${this.apiPath}/${pk}?${queryString}`, obj, {}).toPromise();
+    return this.http.put<any>(`${this.apiPath}/${pk}?${queryString}`, obj, {}).toPromise();
```

Comment:
I think that facility edit also calls this method, and the API is expecting POST there

Reply:
That code was removed a little while ago - facilities are on their own endpoint now.

Reply:
Ah, I guess I'm looking at an old code path